src/Blocks/components/sorting/assets-admin/sorting.js

```diff
@@ -0,0 +1,56 @@
+import Sortable from 'sortablejs';
+
+export class Sorting {
+	constructor(options) {
+		this.selector = options.selector;
+		this.trigger = options.trigger;
+		this.update = options.update;
+
+		this.CLASS_ACTIVE = 'is-active';
+
+		this.sortable = null;
+		this.container = null;
+
+		this.triggerElement = document.querySelector(this.trigger);
+		this.updateElement = document.querySelector(this.update);
+	}
+
+	init = () => {
+		this.triggerElement.addEventListener('click', this.onClickTrigger, true);
+		this.updateElement.addEventListener('click', this.onClickUpdate, true);
+	};
+
+	// Handle form submit and all logic.
+	onClickTrigger = (event) => {
+		event.preventDefault();
+		this.triggerElement.style.display = "none";
+		this.updateElement.style.display = "block";
```

Comment:
The strings here could be written as `this.DISPLAY_STYLE_NONE = 'none'` and `this.DISPLAY_STYLE_BLOCK = 'block'` in the constructor respectively and then used here and in the `onClickUpdate` method (code cleanup).

Reply:
fixed